### PR TITLE
#68 Show sections list at home

### PR DESCRIPTION
**Pull request by rafaparadela**
This PR shows the currently registered sections at content module.

@raulraja Could you review please? Thank you.

### CODE REVIEW

File: scala-exercises-server/app/views/templates/calltosection.scala.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@(section: Section)(implicit request: RequestHeader)
```
Comment (raulraja) [inline scala-exercises-server/app/views/templates/calltosection.scala.html]: I think this should be just `section.scala.html`

File: scala-exercises-server/app/views/templates/home.scala.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@(user: Option[shared.User] = None, redirectUrl: Option[String] = Some("#"))(implicit request: RequestHeader)
+@(user: Option[shared.User] = None, sections: List[models.Section] = List.empty, redirectUrl: Option[String] = Some("#"))(implicit request: RequestHeader)
```
Comment (raulraja) [inline scala-exercises-server/app/views/templates/home.scala.html]: `Nil` instead of `List.empty` is more extended

**Comment by rafaparadela** [comment]
@raulraja All the comments have been addressed. Is good to be merged?

**Comment by raulraja** [comment]
LGTM

**Pull request merged**